hawc/apps/materialized/management/commands/create_materialized_views.py

```diff
@@ -0,0 +1,28 @@
+from django.apps import apps
```

Comment:
An idea - remove create and delete, and just keep refresh? The refresh will create if needed.  I can't think of a use-case for delete, unless you can?

Reply:
I find it useful as a shorthand for recreating mvs when I'm making changes to the sql during development, but I suppose outside of that the delete command doesn't have much use. I can always do it manually or revert migrations but I thought to add it for completeness / this one use-case.